ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/methods/engine/EngineNewPayload.java

```diff
@@ -201,17 +201,6 @@ public JsonRpcResponse syncResponse(final JsonRpcRequestContext requestContext)
       return respondWith(reqId, blockParam, null, SYNCING);
     }
 
-    // TODO: post-merge cleanup
-    if (!mergeCoordinator.latestValidAncestorDescendsFromTerminal(newBlockHeader)) {
-      mergeCoordinator.addBadBlock(block);
-      return respondWithInvalid(
-          reqId,
-          blockParam,
-          Hash.ZERO,
-          INVALID,
-          newBlockHeader.getHash() + " did not descend from terminal block");
-    }
```

Comment:
just a comment - this reflects the perspective that we do whatever the CL tells us to do and do not second guess it.  I think doing this is consistent with that view